ocspresponder.go

```diff
@@ -1,6 +1,44 @@
 package main
 
+import (
+	"encoding/hex"
+	"github.com/cloudflare/cfssl/certdb"
+	"golang.org/x/crypto/ocsp"
+)
+
+// CertDbSource ... TODO
 // An OCSP response source backed by certdb. See
 // cfssl.ocsp.responder.Source.
+// NOTE: implements the cfssl.ocsp.responder.Source interface
 type CertDbSource struct {
+	// Write an implementation of ocsp.responder.Source that fetches an OCSP
+	// response from an instance of certdb
+	Accessor certdb.Accessor
+}
+
+// Response ... TODO
+func (src CertDbSource) Response(req *ocsp.Request) ([]byte, bool) {
+	if req != nil {
```

Comment:
Of course, in a language like Rust you can just use `try!` or `?` instead of tediously typing this out every time :P